src/app/components/SearchResults/ResultsListItem.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Link } from 'react-router';
+import PropTypes from 'prop-types';
+import ResultsRow from './ResultsRow';
+
+/**
+ * ResultsList presents search results as a "grouped" list of books
```

Comment:
Copy/Paste error here. Should fix this comment and perhaps the comment on `ResultsList.jsx` as well.

Reply:
fixed (removed)